package.json

```diff
@@ -2,7 +2,7 @@
     "name": "persistor",
     "description": "A subclass of supertype that serializes to and reconstitutes from mongodb",
     "homepage": "https://github.com/haven-life/persistor",
-    "version": "2.9.1",
+    "version": "2.10.0",
```

Comment:
this seems more like a bug fix than a feature added...should we bump patch version instead?

Reply:
@nsaenz7 - I want to make sure that we use this version after testing all apps.